### PR TITLE
Attempt at multiline action

### PR DESCRIPTION
**Pull request by another-rex**
Update the github actions to pass arguments in a multi line fashion to allow much more customisability.

### CODE REVIEW

File: cmd/osv-reporter/main.go [hunk 1]
```diff
@@ -16,14 +17,24 @@ import (
 
 var (
 	// Update this variable when doing a release
-	version = "1.3.5"
+	version = "1.4.0"
```
Comment (oliverchang) [inline cmd/osv-reporter/main.go:20]: Can we use that new internal package you added for this?

**Review by oliverchang — APPROVED** [review]
Nice! I assume for the scanner the existing splitting in the bash wrapper still works?

File: cmd/osv-reporter/main.go (hunk 1 above)
Comment (another-rex) [reply cmd/osv-reporter/main.go:20]: That's still in the SARIF branch, which hasn't been merged, I'll update it in that branch so this variable uses the new internal package.

**Comment by another-rex** [comment]
> Nice! I assume for the scanner the existing splitting in the bash wrapper still works?

Yep, that still works.

**Pull request merged**